docs/introduction/Car-registry-tutorial.rst

```diff
@@ -401,9 +396,9 @@ Request class shall have appropriate setters and getters for all class members.
     "boxId": "d59f80b39d24716b4c9a54cfed4bff8e6f76597a7b11761d0d8b7b27ddf8bd3c"
  }
         
-Few notes: setter’s input parameter could have a different type than set class member. It allows us to make all necessary conversation in setters.
+A few notes: setter’s input parameter could have a different type than set class member. It allows us to make all necessary conversation (conversion?) in setters.
 
-* Define response for Car creation transaction, the result of transaction shall be defined by implementing SuccessResponse interface with class members which shall be returned as API response, all members shall have properly set getters, also response class shall have proper annotation ``@JsonView(Views.Default.class)`` thus jackson library is able correctly represent response class in JSON format. In our case, we expect to return transaction bytes, so response class is next:
+* Define response for Car creation transaction, the result of transaction shall be defined by implementing SuccessResponse interface with class members which shall be returned as API response, all members shall have properly set getters, also response class shall have proper annotation ``@JsonView(Views.Default.class)`` thus the Jackson library is able to correctly represent response class in JSON format. In our case, we expect to return transaction bytes, so response class is next:
```

Comment:
@zennermac 

* Define the response for the car creation transaction, the result of transaction shall be defined by implementing the SuccessResponse interface with the class members. Class members will be returned as an API response. All members will have properly set getters and the response class will have proper annotation ``@JsonView(Views.Default.class)`` thus the Jackson library is able to correctly represent the response class in JSON format. In our case, we expect to return transaction bytes. The response class is next: